modules/nw-infw-operator-deploying.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-infw-operator-deploying_{context}"]
+= Deploying Ingress Node Firewall Operator
+
+.Prerequisite
+* The Ingress Node Firewall Operator is installed.
+
+.Procedure
+
+To delploy the Ingress Node Firewall Operator, create a `IngressNodeFirewallConfig` custom resource that will deploy the Operator's daemon set. You can deploy one or multiple `IngressNodeFirewall` CRDs to nodes by applying firewall rules.
```

Comment:
```suggestion
To deploy the Ingress Node Firewall Operator, create a `IngressNodeFirewallConfig` custom resource that will deploy the Operator's daemon set. You can deploy one or more `IngressNodeFirewall` CRDs to nodes by applying firewall rules.
```
Spelling

Reply:
@JoeAldinger This issue was not fixed. See [GH55291](https://github.com/openshift/openshift-docs/issues/55291#issuecomment-1407615620). I created a PR to fix it.